### PR TITLE
[AUTH-6] Use auth-service endpoints for auth-related requests

### PR DESCRIPTION
**Pull request by aivopaas**
## Description

-   Switched from old authentication endpoints to the new ones in auth-service

## How to Test

-   Use the Avatar Creator Wizard functionality and verify it works:
    -   Log in with email code
    -   Sign up when clicking Next in the editor

## Checklist

-   [ ] Tests written or updated for the changes.
-   [ ] Documentation is updated.
-   [ ] Changelog is updated.

[AUTH-6]: https://ready-player-me.atlassian.net/browse/AUTH-6?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Runtime/AvatarCreator/Scripts/WebRequests/AuthAPIRequests.cs [hunk 1]
```diff
@@ -60,23 +68,23 @@ public async Task SendCodeToEmail(string email, string userId = "",CancellationT
...
+                { AuthConstants.CODE, code }
             };
             if (userIdToMerge != null)
             {
                 body.Add(AuthConstants.USER_ID, userIdToMerge);
             }
             var payload = AuthDataConverter.CreatePayload(body);
 
-            var response = await webRequestDispatcher.SendRequest<ResponseText>($"{rpmAuthBaseUrl}/auth/login", HttpMethod.POST, headers, payload, ctx:cancellationToken);
+            var response = await webRequestDispatcher.SendRequest<ResponseText>($"{rpmAuthBaseUrl}auth/login", HttpMethod.POST, headers, payload, ctx:cancellationToken);
```
Comment (Kristokarp) [inline Runtime/AvatarCreator/Scripts/WebRequests/AuthAPIRequests.cs:87]: does this login payload now have the appName sent as well?

Comment (aivopaas) [reply Runtime/AvatarCreator/Scripts/WebRequests/AuthAPIRequests.cs:87]: 🤔It does not. I wonder how the  editor worked. Maybe it had a user to merge and that one had the app authorized. Have to check it again.

Comment (Kristokarp) [reply Runtime/AvatarCreator/Scripts/WebRequests/AuthAPIRequests.cs:87]: here it came from subdomain before probably
`string.Format(Env.RPM_SUBDOMAIN_BASE_URL, domain);`

Comment (aivopaas) [reply Runtime/AvatarCreator/Scripts/WebRequests/AuthAPIRequests.cs:87]: I think it worked when I tested with my changes as well but will go over it again anyway

File: Runtime/AvatarCreator/Scripts/Data/CreatedUser.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+
+namespace ReadyPlayerMe.AvatarCreator
+{
+    public struct CreatedUser
+    {
+
```
Comment (HarrisonHough) [inline Runtime/AvatarCreator/Scripts/Data/CreatedUser.cs:9]: minor thing but no need for extra space here

File: Runtime/AvatarCreator/Scripts/Data/CreatedUser.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
```
Comment (HarrisonHough) [inline Runtime/AvatarCreator/Scripts/Data/CreatedUser.cs:2]: Seems none of these using statements are required. They can be removed.
image

**Review by HarrisonHough — APPROVED** [review]
A few minor things in the comments, otherwise all good. Nice work

**Pull request merged**